src/ui/liferea_shell.c

```diff
@@ -1161,7 +1159,9 @@ liferea_shell_restore_state (const gchar *overrideWindowState)
 		gtk_container_child_set (GTK_CONTAINER (liferea_shell_lookup ("wideViewPane")), liferea_shell_lookup ("wideViewItems"),
 			"resize", TRUE, NULL);
 	}
-	
+
```

Comment:
Cannot merge this. Window position is only safe when set from within the idle callback.

Reply:
Ok, I understand.